proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -41,6 +41,10 @@
     @ConfigKey("online-mode")
     private boolean onlineMode = true;
 
+    @Comment("Enables compatibility with the PROXY protocol.")
```

Comment:
PROXY -> HAPROXY

Reply:
PROXY is correct here, given that it's talking about the actual protocol itself and not a specific implementation